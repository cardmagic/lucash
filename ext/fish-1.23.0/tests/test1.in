#
#Test aliases loops and conditionals
#

for i in 1 2 #Comment on same line as command
#Comment inside loop 
    for j in a b
		#Double loop
        echo $i$j
	end;
end

# Simple alias tests

function foo 
    echo >foo.txt $argv
end

foo hello

cat foo.txt |read foo

if test $foo = hello;
  echo Test 2 pass
else
  echo Test 2 fail
end

function foo 
    printf 'Test %s' $1; echo ' pass'
end

foo 3 

for i in Test for continue break and switch builtins problems;
	switch $i
		case Test
			printf "%s " $i 
		case "f??"
			printf "%s " 3
		case "c*"
			echo pass
		case break
			continue
			echo fail
		case and
			break
			echo fail
		case "*"
			echo fail
	end
end

