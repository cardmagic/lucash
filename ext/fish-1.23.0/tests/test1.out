1a
1b
2a
2b
Test 2 pass
Test  pass
Test 3 pass
